Sources/Sentry/SentryThreadInspector.m

```diff
@@ -12,6 +16,36 @@
 
 @end
 
+typedef struct {
+    SentryCrashThread thread;
+    SentryCrashStackEntry stackEntries[100];
```

Comment:
You have the magic number `100` scattered around this file. It might be a good idea to use a `define` for that.